joml-geometry/src/test/java/org/terasology/joml/geom/IntersectionfTest.java

```diff
@@ -180,69 +180,69 @@ public static void testPolygonRay() {
         Vector2f p = new Vector2f();
         float[] verticesXY = { 0, 0, 1, 0, 1, 1, 0, 1 };
         assertEquals(3, Intersectionf.intersectPolygonRay(verticesXY, -1, 0.5f, 1, 0, p));
-        TestUtil.assertVector2fEquals(new Vector2f(0, 0.5f), p, 1E-6f);
+        assertEquals(new Vector2f(0, 0.5f), p, 1E-6f);
```

Comment:
Using the scientific constant here is actually way shorter and concise than using the constants defined below :thinking:  what do we need them for? Should they be used here instead?